scripts/ci/test/test-ci.sh

```
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -euo pipefail
+
+script_dir=$(cd -- $(dirname -- "${BASH_SOURCE[0]}") && pwd)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)